### PR TITLE
Change robots

### PR DESCRIPTION
**Pull request by Bennables**
it will change robots on one line. YAY

### CODE REVIEW

File: src/main/java/frc/robot/Constants.java [hunk 1]
```diff
@@ -26,6 +27,13 @@
  * constants are needed, to reduce verbosity.
  */
 public final class Constants {
+  public record RobotSetup(String name, String canivore_name, GyroType gyroType) {}
```
Comment (RobertTheBlair) [inline src/main/java/frc/robot/Constants.java:30]: Other things we could add here are wheelbase, or other physical robot setup values.

Comment (Bennables) [reply src/main/java/frc/robot/Constants.java:30]: We'll do this later because we have to make photonvision, i'll add this as an issue.

**Review by jessicarosenthal902 — APPROVED** [review]
Changes look good to me. And they were tested on all robots.

**Comment by RobertTheBlair** [comment]
@Bennables probably time to merge this

**Pull request merged**